ibtk/include/ibtk/FEDataManager.h

```diff
@@ -220,6 +232,22 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
         bool use_nodal_quadrature;
     };
 
+    /*!
+     * \brief Struct WorkloadSpec encapsulates the parameters used to
+     * calculate workload estimates (i.e., the input to the load balancing
+     * algorithm).
+     */
+    struct WorkloadSpec
+    {
+        /// Whether or not to clear the workload index arrays at the start of
+        /// the calculation. This should be true unless some other object is
+        /// also adding values into the workload estimate.
+        bool clear_estimate = true;
+
+        /// The multiplier applied to each quadrature point.
+        double q_point_weight = 2.0;
```

Comment:
Is there a way for users to set this value?

Reply:
(I mean in input files, not programatically.)

Reply:
Yes; its in `IBFEMethod`.